run-tests.sh

```diff
@@ -2,13 +2,15 @@
 # -*- coding: utf-8 -*-
 #
 # This file is part of Invenio.
-# Copyright (C) 2015-2018 CERN.
+# Copyright (C) 2015-2020 CERN.
 #
 # Invenio is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
-pydocstyle invenio_records_rest tests docs && \
-isort -rc -c -df && \
-check-manifest --ignore ".travis-*" && \
-sphinx-build -qnNW docs docs/_build/html && \
-python setup.py test
+docker-services-cli services up es postgresql redis
```

Comment:
This would require docker-services-cli to be installed right? Shouldn't this be a test dependency in ``setup.py`` then, or you plan to install it via ``pytest-invenio``?

Reply:
Basically, what I'm looking for is that the following would still work:

```
git clone ...
cd invenio-<module>
pip install -e .[all]
./run-tests.sh
```

Reply:
and something like this for running locally with something else:

```
env ES_VERSION=ES5_LATEST ./run-tests.sh
```

Reply:
Should come with pytest-invenio or be a test dependency yes. My thought was with `pytest-invenio`